src/Sentry.DiagnosticSource/Internals/DiagnosticSource/SentrySqlListener.cs

```diff
@@ -41,8 +41,8 @@ private enum SentrySqlSpanType
         internal const string SqlDataWriteCommandError = "System.Data.SqlClient.WriteCommandError";
         internal const string SqlMicrosoftWriteCommandError = "Microsoft.Data.SqlClient.WriteCommandError";
 
-        private IHub _hub { get; }
-        private SentryOptions _options { get; }
+        private IHub _hub;
+        private SentryOptions _options;
```

Comment:
These can be made `readonly`

Reply:
yeah by removing the `{ get; }` the property will no longer be read only